### PR TITLE
Lk biccn reference descript

### PR DESCRIPTION
**Pull request by ekiernan**
Created reference documentation for BICCN-related pipelines.

### CODE REVIEW

File: website/docs/Pipelines/Optimus_Pipeline/README.md [hunk 1]
```diff
@@ -90,7 +90,7 @@ The example configuration files also contain metadata for the reference files, d
 | input_name_metadata_field | Optional string describing, when applicable, the metadata field containing the input_name. | NA |
 | annotations_gtf | Cloud path to the GTF containing gene annotations used for gene tagging (must match GTF in STAR reference). | NA |
 | chemistry | Optional string description of whether data was generated with 10x v2 or v3 chemistry. Optimus validates this string. If the string does not match one of the optional strings, the pipeline will fail. You can remove the checks by setting "force_no_check = true" in the input JSON | "tenX_v2" (default) or "tenX_v3". |
```
Comment (kayleemathews) [inline website/docs/Pipelines/Optimus_Pipeline/README.md:92]:
```suggestion
| chemistry | Optional string describing whether data was generated with 10x v2 or v3 chemistry. Optimus validates this string. If the string does not match one of the optional strings, the pipeline will fail. You can remove the checks by setting "force_no_check = true" in the input JSON | "tenX_v2" (default) or "tenX_v3". |
```

File: website/docs/Pipelines/Optimus_Pipeline/README.md [hunk 2]
```diff
@@ -90,7 +90,7 @@ The example configuration files also contain metadata for the reference files, d
...
-| counting_mode | String description of whether data is single-cell or single-nucleus. | "sc_rna" or "sn_rna" |
+| counting_mode | String description of whether data is single-cell or single-nucleus. Single-cell mode counts reads aligned to the gene transcript, where as single-nucleus counts whole transcript to account for nuclear pre-mRNA.| "sc_rna" or "sn_rna" |
```
Comment (kayleemathews) [inline website/docs/Pipelines/Optimus_Pipeline/README.md:93]:
```suggestion
| counting_mode | String describing whether data is single-cell or single-nucleus. Single-cell mode counts reads aligned to the gene transcript, whereas single-nucleus counts whole transcript to account for nuclear pre-mRNA. | "sc_rna" or "sn_rna" |
```

File: website/docs/Pipelines/Optimus_Pipeline/consortia-processing.md [hunk 3]
```diff
@@ -115,5 +115,22 @@ For example code showing how to link a contributor matrix to a DCP project matri
...
 
+## Brain Initiative Cell Census Network Processing
+The Optimus pipeline supports data processing for the [BRAIN Initiative Cell Census Network (BICCN)](https://biccn.org/). An overview of the BICCN pipeline resources is available on the BICCN's [Pipelines page](https://biccn.org/tools/biccn-pipelines).
+
+### Optimus reference files for BICCN data processing
+The BICCN 2.0 Whole Mouse Brain Working Group uses the Ensembl GRCm38 reference for alignment and a modified GTF for gene annotation (see table below). All Optimus pipeline reference inputs were created with the [BuildIndices workflow](https://github.com/broadinstitute/warp/tree/master/pipelines/skylab/build_indices).
+
+BICCN processes single-nucleus data, which is enriched in pre-mRNAs containing introns. To account for this, the BuildIndices workflow uses the `BuildStarSingleNucleus` task to add intron annotations to the GTF with a custom [python script](https://github.com/broadinstitute/warp/blob/develop/dockers/skylab/snss2-build-indices/add-introns-to-gtf.py). The GTF contains all annotations for any gene_id that has at least one transcript. This reduces the number of genes in the GTF to \~32,000.
```
Comment (kayleemathews) [inline website/docs/Pipelines/Optimus_Pipeline/consortia-processing.md:124]:
```suggestion
BICCN processes single-nucleus data, which is enriched in pre-mRNAs containing introns. To account for this, the BuildIndices workflow uses the `BuildStarSingleNucleus` task to add intron annotations to the GTF with a custom [python script](https://github.com/broadinstitute/warp/blob/master/dockers/skylab/snss2-build-indices/add-introns-to-gtf.py). The GTF contains all annotations for any `gene_id` that has at least one transcript. This reduces the number of genes in the GTF to \~32,000. 
```

File: website/docs/Pipelines/Smart-seq2_Single_Nucleus_Multi_Sample_Pipeline/consortia-processing.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+### Multi-snSS2 reference files for BICCN data processing
+The BICCN 2.0 Whole Mouse Brain Working Group uses the Ensembl GRCm38 reference for alignment and a modified GTF for gene annotation (see table below). All Multi-snSS2 pipeline reference inputs were created with the [BuildIndices workflow](https://github.com/broadinstitute/warp/tree/master/pipelines/skylab/build_indices).
+
+ BICCN processes single-nucleus data, which is enriched in pre-mRNAs containing introns. To account for this, the Multi-snSS2 workflow counts reads that map to both exonic and intronic regions (any part of a contig that is not exonic nor intergenic). The BuildIndices workflow uses the `BuildStarSingleNucleus` task to add intron annotations to the GTF with a custom [python script](https://github.com/broadinstitute/warp/blob/develop/dockers/skylab/snss2-build-indices/add-introns-to-gtf.py). These annotations enable intron counting with the [featureCounts](http://subread.sourceforge.net/) software.
```
Comment (kayleemathews) [inline website/docs/Pipelines/Smart-seq2_Single_Nucleus_Multi_Sample_Pipeline/consortia-processing.md:13]:
```suggestion
 BICCN processes single-nucleus data, which is enriched in pre-mRNAs containing introns. To account for this, the Multi-snSS2 workflow counts reads that map to both exonic and intronic regions (any part of a contig that is not exonic nor intergenic). The BuildIndices workflow uses the `BuildStarSingleNucleus` task to add intron annotations to the GTF with a custom [python script](https://github.com/broadinstitute/warp/blob/master/dockers/skylab/snss2-build-indices/add-introns-to-gtf.py). These annotations enable intron counting with the [featureCounts](http://subread.sourceforge.net/) software. 
```

File: website/docs/Pipelines/Smart-seq2_Single_Nucleus_Multi_Sample_Pipeline/consortia-processing.md [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+
+### Multi-snSS2 reference files for BICCN data processing
+The BICCN 2.0 Whole Mouse Brain Working Group uses the Ensembl GRCm38 reference for alignment and a modified GTF for gene annotation (see table below). All Multi-snSS2 pipeline reference inputs were created with the [BuildIndices workflow](https://github.com/broadinstitute/warp/tree/master/pipelines/skylab/build_indices).
+
+ BICCN processes single-nucleus data, which is enriched in pre-mRNAs containing introns. To account for this, the Multi-snSS2 workflow counts reads that map to both exonic and intronic regions (any part of a contig that is not exonic nor intergenic). The BuildIndices workflow uses the `BuildStarSingleNucleus` task to add intron annotations to the GTF with a custom [python script](https://github.com/broadinstitute/warp/blob/develop/dockers/skylab/snss2-build-indices/add-introns-to-gtf.py). These annotations enable intron counting with the [featureCounts](http://subread.sourceforge.net/) software. 
+
+ The custom GTF contains all annotations for any gene_id that has at least one transcript. This reduces the number of genes in the GTF to \~32,000.
```
Comment (kayleemathews) [inline website/docs/Pipelines/Smart-seq2_Single_Nucleus_Multi_Sample_Pipeline/consortia-processing.md:15]:
```suggestion
 The custom GTF contains all annotations for any `gene_id` that has at least one transcript. This reduces the number of genes in the GTF to \~32,000. 
```

**Review by kayleemathews — APPROVED** [review]
Looks good! Just a few very minor things I would change.

**Comment by ekiernan** [comment]
Thanks @kayleemathews for the review!

File: website/docs/Pipelines/Smart-seq2_Single_Nucleus_Multi_Sample_Pipeline/consortia-processing.md [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+| --- | --- | --- | --- |
+| `annotations_gtf` | gs://gcp-public-data--broad-references/mm10/v0/single_nucleus/modified_gencode.vM23.primary_assembly.annotation.gtf | https://ftp.ebi.ac.uk/pub/databases/gencode/Gencode_mouse/release_M23/gencode.vM23.annotation.gtf.gzf | Modified GENCODE GTF including intron annotations that can be used for intron counting with featureCounts. |
+| `genome_ref_fasta` | gs://gcp-public-data--broad-references/mm10/v0/single_nucleus/modified_mm10.primary_assembly.genome.fa | https://ftp.ebi.ac.uk/pub/databases/gencode/Gencode_mouse/release_M23/GRCm38.p6.genome.fa.gz | FASTA file used to create the STAR reference files. |
+| `tar_star_reference` | gs://gcp-public-data--broad-references/mm10/v0/single_nucleus/star/modified_star_2.7.9a_primary_gencode_mouse_vM23.tar | NA — built with the BuildIndices workflow. | Reference files used for alignment with STAR. |
+| `adapter_list` | gs://hca-dcp-sc-pipelines-test-data/adapters/Illumina_adapters_list.fa | See Illumina's overview on [adapter sequences](https://support.illumina.com/bulletins/2016/12/what-sequences-do-i-use-for-adapter-trimming.html). | List of adapter sequences used for trimming. |
+
+
```
Comment (gbggrant) [inline website/docs/Pipelines/Smart-seq2_Single_Nucleus_Multi_Sample_Pipeline/consortia-processing.md:26]: Unnecessary trailing lines?

**Review by khajoue2 — APPROVED** [review]

**Pull request merged**